src/styles/_catalog.scss

```diff
@@ -1,9 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
```

Comment:
1 - use repeat for this purpose
2 - why auto? use card width here (and create variable for it), then you'll be able to remove width from card